gltf/write.cpp

```diff
@@ -602,6 +602,7 @@ void writeImage(std::string& json, std::vector<BufferView>& views, const cgltf_i
 			if (readFile(full_path.c_str(), img_data))
 			{
 				std::string encoded;
+				mime_type = image.mime_type ? image.mime_type : inferMimeType(image.uri);
```

Comment:
Ah, makes sense - I forgot about this. Can this be moved to the beginning of the function, perhaps as a `else if (image.uri)` after the inferMimeType is being called right now? That will make mime_type more consistently present throughout the rest of the function.